### PR TITLE
interactive bots: Run the message_info bot for fredfishgames.

### PR DESCRIPTION
**Pull request by fredfishgames**

### CODE REVIEW

File: interactive-bots/message_info/fredfishgames/notes.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Task went fine, apart from accidentally putting quotation marks around `content` and `original_sender`, which meant some messages were not sent, or content was not sent.
```
Comment (tommyip) [inline interactive-bots/message_info/fredfishgames/notes.md]: Can you line wrap at 79 characters? Doing this allows the output of  `git diffs` to be readable in terminal.

**Comment by fredfishgames** [comment]
Hi
I have renamed all the files, and corrected the line length of `notes.md`

**Comment by roberthoenig** [comment]
@fredfishgames nice work, almost done! Can you retake the `chat.png` screenshot so that we see these messages were sent in the `Verona` stream? As a general note for future tasks, it is always better to show too much than too little on screenshots :)

**Comment by fredfishgames** [comment]
I've updated the screenshot

**Comment by roberthoenig** [comment]
Great work @fredfishgames! Approving and merging.

**Pull request merged**